### PR TITLE
fix(core): remove timeline node tab index

### PR DESCRIPTION
**Pull request by mikerodonnell89**
part of #8897

### CODE REVIEW

File: libs/core/src/lib/dynamic-page/dynamic-page.component.scss [hunk 1]
```diff
@@ -27,7 +27,7 @@
     height: 100%;
 
     .fd-bar--floating-footer {
-        z-index: 2;
+        z-index: 6;
```
Comment (platon-rov) [inline libs/core/src/lib/dynamic-page/dynamic-page.component.scss:30]: Looks like it's not related to timeline.

**Review by platon-rov — CHANGES_REQUESTED** [review]

File: libs/core/src/lib/dynamic-page/dynamic-page.component.scss (hunk 1 above)
Comment (mikerodonnell89) [reply libs/core/src/lib/dynamic-page/dynamic-page.component.scss:30]: Wrong branch, thanks!
